@theme/styles.css

```diff
@@ -159,7 +159,7 @@ ul.nav.navbar-nav {
   --color-geekblue-10: #030852; */
 
   /* Dark mode colors by default */
-  --color-primary-bg: var(--color-gray-9);
+  --color-primary-bg: var(--color-gray-10);
```

Comment:
for transparency --color-gray-10 is a reference to color-gray-9 so there will be no change in the color for those vars